crowbar_framework/lib/crowbar/upgrade.rb

```diff
@@ -92,6 +94,7 @@ def knife_files
 
       roles_path = knife_path.join("roles")
       roles_path.children.each do |file|
+        logger.debug "Upgrading #{file.to_s}"
```

Comment:
Lint/StringConversionInInterpolation: Redundant use of `Object#to_s` in interpolation. (https://github.com/bbatsov/ruby-style-guide#no-to-s)